src/source/spidermonkey.rs

```diff
@@ -36,6 +36,7 @@ impl SpiderMonkey {
     fn parse_script_output(&self, script: &str) -> Result<JSON, Error> {
         info!(target: "SpiderMonkey", "Launching script {}", script);
         let output = Command::new(&*self.bin_path)
+            .arg("--max_old_space_size=4096")
```

Comment:
That one doesn't make sense, since it doesn't launch Node. Still, I should remove the module anyway :)